### DISCUSSION TITLE
英文文本排版问题

### DISCUSSION THREAD

**Discussion by AlexYXB** [Tencent/libpag#486] [General] [answered]
AE中设置一个文字区域，实际替换文字内容的时候，如果是英文内容会出现截断单词换行的情况，请问可以通过什么方式避免吗？

**Comment by domchen** [comment, marked as answer]
word wrap 跟从右往左排的需求类似，都属于高级文本排版能力，这块要整体支持起来复杂度会非常高，目前暂时还没有这块的排期。但是有统一的解法的。就是通过图片去替换而不是直接换文本。文本可以用系统的组件进行渲染出想要的排版截图，然后PAG素材里放个占位图用来替换就行。目前不少出海需求的项目也是在用这种解法实现替换。
